contracts/cw721-base/src/contract_tests.rs

```diff
@@ -194,6 +194,7 @@ fn transferring_nft() {
         Response::new()
             .add_attribute("action", "transfer_nft")
             .add_attribute("sender", "venus")
```

Comment:
Language is a bit confusing her now... we have `sender`, `ricpient`, and `from`...